meta/static/css/footer.css

```diff
@@ -50,19 +50,24 @@
     padding: 0.3rem 0.7rem 0 0.7rem; 
 }
 
-@media (max-width: 660px) {
+/* @media (max-width: 760px) {
     #footerbase {
         padding-bottom: 1rem;
     }
 
     #footerbase > div:last-child {
         flex-direction: column;
     }
-}
+} */
```

Comment:
Is it necessary to keep this section commented out?

Reply:
No, my mistake. I should've deleted it.